nexus/src/app/background/tasks/region_snapshot_replacement_step.rs

```diff
@@ -413,12 +413,6 @@ impl BackgroundTask for RegionSnapshotReplacementFindAffected {
         opctx: &'a OpContext,
     ) -> BoxFuture<'a, serde_json::Value> {
         async move {
-            let log = &opctx.log;
-            info!(
```

Comment:
These were the logs that were too noisy?

Reply:
Yeah, see https://github.com/oxidecomputer/omicron/pull/6323